test/integration/load/load_test.go

```diff
@@ -40,6 +40,13 @@ var noopDeploymentMap = map[string]string{
 	"linux":   manifestDir + "/noop-deployment-linux.yaml",
 }
 
+// This map is used exclusively for TestLoad. Windows is expected to take 10-15 minutes per iteration.
+// Will change this as scale testing results are verified. This will ensure we keep a standard performance metric.
```

Comment:
Does this we will verify after merging or before ?

Reply:
After, as an ongoing process. Currently the only performance metric that I know of is for windows clusters and timed accordingly. Have not tested for linux or cilium specifically.